### ISSUE TITLE
Locked out of magic system entirely

### ISSUE THREAD

**Issue by zammorac** [Mithion/ArsMagica2#234]
I was playing on a LAN server as a guest and decided to store stuff in a Rift spell. When the server shut down for the night it appears to have erased everything in the Rift. Normally that wouldn't be too horrible but this time I put my arcane compendium in it. When the server reopened the next day I had no mana bar or magic level. Again this seemed fixable except now I can't even make new arcane compendiums, it does the animation but stays as a book. Everyone else on the server can still use magic and make compendiums so I believe this has to do with the disappearance of my arcane compendium in an unexpected way.

TLDR Storing an arcane compendium in a Rift when you are a guest on a LAN server causes it to be deleted on server reset and locks you out of the magic system entirely.
